lib/ansiblelint/rules/__init__.py

```diff
@@ -36,7 +36,10 @@ def verbose(self) -> str:
 
     @staticmethod
     def unjinja(text):
-        return re.sub(r"{{[^}]*}}", "JINJA_VAR", text)
+        text = re.sub(r"{{[^}]*}}", "JINJA_EXPRESSION", text)
```

Comment:
Should this not replace jinja looking like "{{ test_var | combine({'key': 'value'}) }}" with "JINJA_EXPRESSION" for some reason?

Because the current regex won't do that.